.github/workflows/k8s-tests.yml

```diff
@@ -4,6 +4,9 @@ on:
   pull_request:
     branches:
       - main
+  push:
```

Comment:
This will run 2 times

Reply:
@zhilyaev Nope - it didn't run at all. https://github.com/helmwave/helmwave/actions?query=branch%3Amain++